AUTHORS

```diff
@@ -0,0 +1,7 @@
+Brad Nelson <bnelsj@gmail.com>
+Brad Nelson <brad@phasegenomics.com>
+EC2 Default User <ec2-user@ip-172-31-25-134.us-west-2.compute.internal>
```

Comment:
Is it necessary to have the default ec2-user be an author? In theory anyone can be that user